### PR TITLE
Allow edition of project name on rightSideBar on Wizard

### PR DESCRIPTION
**Pull request by javieraparisivaldes**
- Quick summary of changes
Allow edition of project name on rightSideBar on Wizard
- Which issue does this PR relate to?
#1552

### CODE REVIEW

File: src/client/src/components/ProjectName/index.tsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
+import * as React from "react";
```
Comment (prupipho) [inline src/client/src/components/ProjectName/index.tsx:1]: Parece que está roto, estoy investigando a ver cómo arreglarlo, creo que hay un import equivocado en ProjectOutput/index.tsx : )
https://winappstudio.visualstudio.com/WTS/_build/results?buildId=34666&view=logs&j=275f1d19-1bd8-5591-b06b-07d489ea915a&t=2d9a8608-ccd5-5e55-2855-4c1cfc196e57&l=48

File: src/client/src/components/ProjectName/index.tsx [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+import { useDispatch, useSelector } from "react-redux";
```
Comment (prupipho) [inline src/client/src/components/ProjectName/index.tsx:2]: Project Name ya no pone myApp por defecto
Save To no pone ningún path por defecto
Cuando haces click en browse da error: invalid command used

image

**Comment by prupipho** [comment]
When loading the extension doesn´t set default Project Name and Path:
image

**Review by dgomezc — APPROVED** [review]

**Comment by prupipho** [comment]
review these behaviours: 
- what happens when you type an invalid Project Name and lose focus from that box
- what happens when you select an invalid Page Name on the Right-Side bar

**Pull request merged**